test/testBearingRange2D.jl

```diff
@@ -329,25 +329,32 @@ addVariable!(fg, :x0, Pose2)
 setVal!(fg, :x0, [getPointIdentity(Pose2)])
 
 # Add bearing range measurement between pose and landmark
-p2br = Pose2Point2BearingRange(Normal(0,0.1),Normal(20.0,1.0))
+p2br = Pose2Point2BearingRange(Normal(0,0.1),Normal(20.0,0.1))
 addFactor!(fg, [:x0; :l1], p2br, graphinit=false)
 
 # there should be just one (the bearingrange) factor connected to :l1
 @test length(ls(fg, :x0)) == 1
 # writeGraphPdf(fg)
 
 # check the forward convolution is working properly
-_pts, = predictbelief(fg, :x0, ls(fg, :x0), N=75)
+_pts, = predictbelief(fg, :x0, ls(fg, :x0); N)
```

Comment:
pose should be anywhere on donut ring around the landmark, including at xytheta=(0,0,0)

Reply:
Jip. That’s why i had to change the test. It only tested close to 0,0